options_test.go

```diff
@@ -25,6 +25,108 @@ import (
 	models "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
 )
 
+func TestGetFirecrackerConfig(t *testing.T) {
+	tempFile, err := ioutil.TempFile("", "firectl-test-drive-path")
+	if err != nil {
+		t.Error(err)
+	}
+	defer os.Remove(tempFile.Name())
+	defer tempFile.Close()
+	cases := []struct {
+		opts        *options
+		expectedErr func(error) (bool, error)
+		outConfig   firecracker.Config
+	}{
+		// invalid metadata
```

Comment:
I'd make a `name string` field in the struct and actually put this in the struct.  Then you can use it nicely with subtests.